docs/UsersGuide.asciidoc

```diff
@@ -554,6 +554,25 @@ Steps 4. to 6. can be repeated for further +assert_screen+s/needles without rest
 test.
 
 
+=== Job group editor https://github.com/os-autoinst/openQA/pull/2111[gh#2111] ===
+
+Scenarios are defined as part of a job group. The `Edit job group` button exposes the editor.
+
+By default the scenarios are listed in a table per medium. Changes to the priority are
+applied immediately. Machines can be added by selecting the architecture column and picking
+a machine from the list. Remove scenarios by removing all of their machines. Add new scenarios
+via the blue Plus icon at the top of the table.
+
+The `Edit YAML` button can be used to reveal the YAML editor and migrate a group. After
+saving for the first time, the group can only be configured in YAML. The table view won't
+be shown anymore.
```

Comment:
please don't use abbreviated forms in the official docs (at least this is what I learned to be best practice) so s/won't/will not/. In github comments it's ok though ;)